docs/prerequisites.md

```diff
@@ -0,0 +1,48 @@
+# Prerequisites
+
+## Requirements
+
+- Linux or MacOS (Windows isn't supported at the moment).
+- A [Google Cloud](https://console.cloud.google.com) account.
+- [Packer](https://packer.io) to build images
+- Make to use `Makefile` targets
```

Comment:
Make should be lower case `M`?

Reply:
Yeah, it's capitalized because it's the start of a sentence 😬